content/news/035/index.md

```diff
@@ -353,6 +353,22 @@ _Discussions: [/r/rust_gamedev](https://www.reddit.com/r/rust_gamedev/comments/v
 [wgpu]: https://github.com/gfx-rs/wgpu
 [ggez-issues]: https://github.com/ggez/ggez/issues
 
+### [miniquad]
+
+![miniquad fileopen](fileopen.gif)
+
+[miniquad] is a safe and cross-platform rendering library
+focused on portability and low-end platforms support.
+
+[A java interop write-up][miniquad-java]
```

Comment:
feels like this line should be merged with the next paragraph

Reply:
It feels like this was intended as a subheading (which we usually try to avoid) - I've rephrased the section a bit to fix this.